extensions/git/src/main.ts

```diff
@@ -180,13 +180,8 @@ export async function activate(context: ExtensionContext): Promise<GitExtension>
 	return result;
 }
 
-async function checkGitVersion(_info: IGit): Promise<void> { // {{SQL CARBON EDIT}} - Rename info to _info to prevent error due to unused variable
-	return; /* {{SQL CARBON EDIT}} return immediately
-
-	/*const config = workspace.getConfiguration('git');
-	const shouldIgnore = config.get<boolean>('ignoreLegacyWarning') === true;
-
-
+// @ts-expect-error
+async function checkGitVersion(info: IGit): Promise<void> {
```

Comment:
are we going to start getting the legacy git warning now?  Ideally we won't since it's not relevant to a lot of ADS customers and we already have too many notifications on 1st launch.

Reply:
I commented how the part that calls this function. https://github.com/microsoft/azuredatastudio/pull/10442/files#diff-7c73a37cb94032abbc15f0dc33c88469R81